driver/src/cosp2_test.f90

```diff
@@ -549,7 +548,7 @@ subroutine subsample_and_optics(nPoints, nLevels, nColumns, nHydro, overlap,
          mr_ccice,dtau_c,dtau_s,dem_c,dem_s,fl_lsrainIN,fl_lssnowIN,fl_lsgrplIN,fl_ccrainIN,&
          fl_ccsnowIN
     real(wp),intent(in),dimension(nPoints,nLevels,nHydro) :: reffIN
-    logical,intent(in) :: use_precipitation_fluxes
+    logical,intent(in) :: use_vgrid, use_precipitation_fluxes
```

Comment:
Could you add a comment here explaining what this means?

Reply:
use_vgrid controls the use of model levels (.false.) or an evenly-spaced (.true.) vertical grid for outputs.
@RobertPincus did you mean 'here' in the discussion or a comment in the code itself?

Reply:
@alejandrobodas I meant a comment in the code itself, as `use_vgrid` is maybe less obviously clear than say `use_precipitation_fluxes`

Reply:
@RobertPincus I see, I've added comment as requested.